src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -76,6 +77,7 @@ Profile::Profile(const winrt::guid& guid) :
     _cursorColor{ DEFAULT_CURSOR_COLOR },
     _cursorShape{ CursorStyle::Bar },
     _cursorHeight{ DEFAULT_CURSOR_HEIGHT },
+    _doubleClickDelimiters{ L" /\\" },

```

Comment:
Do we maybe want something more comprehensive to be the defaults? like `" ./\\()\"'-:,.;<>~!@#$%^&*|+=[]{}`~?"` maybe?

Reply:
I'm really glad you said something here. I kinda just chose something and went with it haha.